package.json

```diff
@@ -50,64 +50,59 @@
   "publishConfig": {
     "access": "public"
   },
-  "dependencies": {
-    "@fortawesome/free-brands-svg-icons": "^5.13.0",
-    "@fortawesome/free-solid-svg-icons": "^5.12.1",
-    "@fortawesome/vue-fontawesome": "^0.1.9",
-    "@system76/design": "^5.2.1"
-  },
```

Comment:
These should still be dependencies. Not sure how that got switched.

Reply:
I had to run an `npm update` which is likely what happened here. Can I add these back manually to fix it?

Reply:
@btkostner does this need to be fixed?

Reply:
You should be able to just move these back to `dependencies` and run `npm i`